modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -40,6 +44,7 @@
   $redis_host = undef,
   $email_alert_api_bearer_token = undef,
   $enable_unpublishing_queue_consumer = false,
+  $enable_subscriber_list_update_queue_consumers =  false,
```

Comment:
Extra unnecessary space before the `false` (kind of surprised the linter doesn't complain about that, it's really picky about puppet spacing elsewhere...)